DemoServer/AdditionalLanguages/nodejs/demo/autoIndexes/autoMapReduceIndex/index.js

```diff
@@ -1,34 +1,25 @@
 const { documentStore } = require('../../common/docStoreHolder');
 
 async function run () {
-    /**
-     * TODO:
-     *
-     * please update steps
-     */
-
     //region Demo
-
     const session = documentStore.openSession();
 
+    //region Step_1
     const numberOfEmployeesPerCountry = await session.query({ collection: 'employees' })
+    //endregion
+        //region Step_2    
         .groupBy('Address.Country')
+        //endregion
+        //region Step_3
         .selectKey('Address.Country', 'Country')
         .selectCount('NumberOfEmployees')
+        //endregion
+        //region Step_4
         .orderByDescending('NumberOfEmployees')
+        //endregion
+        //region Step_5
         .all();
-        // [
```

Comment:
We don't have the result classes defined like in Java (what would be put there anyway except a constructor - you can''t define fields or their type in a class anyway in vanilla JS. I think it might be worth leaving this (I put it here on purpose) to show how exactly the results look like. Or maybe put it in a step description or somewhere else?

Reply:
done - added back

Reply:
done - added back